components/disclaimer-modal.vue

```diff
@@ -72,6 +69,23 @@
         this.open = false
       }
     },
+    mounted() {
+      fetch('/docs/user-agreements.md')
```

Comment:
why fetch it?  So it can be changed between releases?

Reply:
because the file wasn't loaded properly as a string otherwise